### ISSUE TITLE
`unknown enum constant` warning during compilation

### ISSUE THREAD

**Issue by tomasulo** [palantir/docker-compose-rule#96]
When I compile a project using docker-compose-rule:0.18.0 this warning appears:

`warning: unknown enum constant ImplementationVisibility.PACKAGE
  reason: class file for org.immutables.value.Value$Style$ImplementationVisibility not found`

Is there a way to fix this?

This is my build configuration:

```
------------------------------------------------------------
Gradle 2.14.1
------------------------------------------------------------

Build time:   2016-07-18 06:38:37 UTC
Revision:     d9e2113d9fb05a5caabba61798bdb8dfdca83719

Groovy:       2.4.4
Ant:          Apache Ant(TM) version 1.9.6 compiled on June 29 2015
JVM:          1.8.0_101 (Oracle Corporation 25.101-b13)
OS:           Linux 3.19.0-32-generic amd64
```

**Comment by iamdanfox** [comment]
This seems to just be a linting warning, but I think it stems from a direct [package visibility annotation][1], which apparently isn't recommended.  More discussion here: https://github.com/immutables/immutables/issues/291, and an example fix here: https://github.com/palantir/dropwizard-web-security

Would you be interested in submitting a PR?

[1]: https://github.com/palantir/docker-compose-rule/blob/4c590d341e1923a1cc966da3ddcf95855e43f421/src/main/java/com/palantir/docker/compose/configuration/ProjectName.java#L29

**Comment by tomasulo** [comment]
Sure, I'll look into it.

iamdanfox <notifications@github.com> schrieb am Do., 25. Aug. 2016, 23:09:
